README.md

```diff
@@ -4,6 +4,77 @@ JS Client library to interact with Supabase Storage.
 
 - Documentation: https://supabase.io/docs/reference/javascript/storage-createbucket
 
+## Quick Start Guide
+
+### Installing the module
+
+``` bash
+npm install @supabase/storage-js
+```
+
+### Connecting to the storage backend
+
+``` bash
+# .env
+SUPABASE_STORAGE_URL=https://<project_ref>.supabase.co/storage/v1
+SUPABASE_SERVICE_KEY=<service_role>
+```
+
+``` js
+// index.hs
+import { SupabaseStorageClient } from '@supabase/storage-js'
+import 'dotenv/config'
```

Comment:
Not everyone would use `dotenv`. So I think just having the constants directly, without using environment variables, would be good enough for an example.

Reply:
Fixed [here](https://github.com/supabase/storage-js/pull/47/commits/f08df8322bf9b89e5b51e2465ea3a8230907633b)